openerp/tools/translate.py

```diff
@@ -366,7 +366,8 @@ def next(self):
                 return self.next()
 
             if not line.startswith('msgid'):
-                raise Exception("malformed file: bad line: %s" % line)
+                _logger.info(_('Malformed file: Bad line: %s') % (line), exc_info=True)
+                raise openerp.exceptions.UserError(_('Malformed file \n\n Bad line: %s') % (line))
             source = unquote(line[6:])
```

Comment:
first point are changed. but last point, we can't import UserError on top given import error. that's why I use openerp.exceptions.UserError(...). Is there any other way?